client/stylesheets/_toCat.scss

```diff
@@ -5195,6 +5201,8 @@ On click the images will be displayed at normal size to complete the effect
 .profileUserFeedbackViewResponse {
     font-size: 1rem;
     border-radius: 4px;
+    width: 80% !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces